src/pages/docs/administration/data/backup-and-restore.md

```diff
@@ -18,7 +18,7 @@ A successful disaster recovery plan for Octopus Deploy requires the ability to r
 :::div{.problem}
 
 **Without your Master Key, backups are useless**
-Sensitive information is encrypted using AES128 with the Master Key as the encryption key. Without this Master Key you will lose your sensitive variables, passwords, and other encrypted data. Make sure you've taken a copy of the key! [Learn more about backing up the Master Key](/docs/security/data-encryption).
+Sensitive information is encrypted using AES-256 with the Master Key as the encryption key. Without this Master Key you will lose your sensitive variables, passwords, and other encrypted data. Make sure you've taken a copy of the key! [Learn more about backing up the Master Key](/docs/security/data-encryption).
```

Comment:
I think this would be fine wording if we'd used AES-256 for years, but given that customers will start reading this as soon as tomorrow, whilst the actual AES change won't reach them for _months_ or even _years_ I feel like we'd be better to qualify it. Perhaps something like

> Sensitive information is encrypted using AES* with the Master Key as the encryption key...
>
> * Octopus Server 2024.4 and newer use AES256 by default but support AES128 for compatibility. Older versions use AES128.

Reply:
Or just don't specify the key length at all; anyone who's interested would likely then follow the link to the data-encryption doc

Reply:
Addressed.